orderer/common/filerepo/filerepo.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package filerepo
+
+import (
+	"bufio"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hyperledger/fabric/common/configtx"
+)
+
+type Repo struct {
+	Options Options
+}
+
+type Options struct {
+	FileSuffix          string
+	TransientFilePrefix string
```

Comment:
There may be some use case for configuring the prefix, but currently it will probably always be `~` so we can probably remove it entirely from the options and leave it as an internal package constant.